handbook/sales/README.md

```diff
@@ -128,6 +128,7 @@ To schedule an [ad hoc meeting](https://www.vocabulary.com/dictionary/ad%20hoc)
  - Use [this calendly link](https://calendly.com/fleetdm/talk-to-a-solutions-consultant) to obtain SC availability.
  - The AE will populate this issue with the appropriate dates for an internal prep meeting as well as the dates for the external prospect meeting.
  - Do not assign the issue. The Director of Solutions Consulting will assign the issue.
+ - Ensure that the product category is defined (Endpoint ops, Device management, or Vulnerability management) in the description of the issue.
```

Comment:
```suggestion
 - Ensure that the product category is defined ("Endpoint ops", "Device management", or "Vulnerability management") in the description of the issue.
```
We can either throw quotes around them or all lower-case, other than that, all good!